logconfig.yaml

```
@@ -24,6 +24,12 @@ loggers:
   urllib3:
     level: WARNING
     propagate: true
+  UnleashClient:
+    level: WARNING
+    propagate: true
+  apscheduler:
+    level: WARNING
+    propagate: false
```

I'm afraid it will not log the any message if `propagate` is set to `false`.

```suggestion
  apscheduler:
    level: WARNING
    propagate: true
```

---

Ah OK, I'll change that back